src/Aspire.Dashboard/Components/Pages/Traces.razor

```diff
@@ -49,64 +49,69 @@
+                                            {
+                                                <div style="margin-top: 8px; margin-bottom: 8px;">
+                                                    @item.ChildContent
+                                                </div>
+                                            }
+                                        </FluentTooltip>
+                                    </OverflowTemplate>
+                                </FluentOverflow>
+                            </TemplateColumn>
+
+                            <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.DurationColumnHeader)]">
+                                <div class="duration-container">
+                                    @if (ViewportInformation.IsDesktop)
+                                    {
+                                        <FluentProgressRing Class="duration-ring"
+                                                            Min="0"
+                                                            Max="@Convert.ToInt32(TracesViewModel.MaxDuration.TotalMilliseconds)"
+                                                            Value="@Convert.ToInt32(context.Duration.TotalMilliseconds)"
+                                                            aria-label="@ControlsStringsLoc[nameof(ControlsStrings.DurationColumnHeader)]"/>
+                                        <span class="trace-duration">
+                                            @DurationFormatter.FormatDuration(context.Duration)
+                                        </span>
+                                    }
+                                    else
+                                    {
                                         @DurationFormatter.FormatDuration(context.Duration)
-                                    </span>
-                                }
-                                else
-                                {
-                                    @DurationFormatter.FormatDuration(context.Duration)
-                                }
-                            </div>
-                        </TemplateColumn>
+                                    }
+                                </div>
+                            </TemplateColumn>
+                        }
 
                         <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.DetailsColumnHeader)]" Class="no-ellipsis">
-                            <FluentAnchor Appearance="Appearance.Lightweight" Href="@DashboardUrls.TraceDetailUrl(context.TraceId)" @onclick:stopPropagation="true">@ControlsStringsLoc[nameof(ControlsStrings.ViewAction)]</FluentAnchor>
+                            <FluentAnchor Appearance="Appearance.Lightweight" Href="@DashboardUrls.TraceDetailUrl(context.TraceId)">@ControlsStringsLoc[nameof(ControlsStrings.ViewAction)]</FluentAnchor>
                         </TemplateColumn>
```

Comment:
Once fluentui-blazor opens items on <kbd>Enter</kbd>, we can hide this column for very narrow screens (or maybe all screens).

Reply:
https://github.com/microsoft/fluentui-blazor/issues/2540

Reply:
Will move this to its own issue as part of #5317.